### ISSUE TITLE
슬랙 앱을 공유하는 3가지 스텝(Distributing Slack apps)

### ISSUE THREAD

**Issue by giyeonkwon** [laranhee/bot.devwithluv.dev#8]
1. Single-workspace apps - **현재 개발된 단계**

2. Apps distributed to multiple workspaces
- 다음 문서 참고: https://api.slack.com/start/distributing/public
- 자세히 더 살펴봐야하지만, 바코에 devWithLuv앱을 설치하기 위해선 바코 collaborator가 어떠한 방법으로 직접 설치를해야함.

3. Apps distributed via the App Directory (GIPHY처럼 App Directory검색을 통해 설치 할수 있음) - 퀄리티와 사용성을 충족해야함
- 다음 문서 참고:https://api.slack.com/start/distributing/directory

**Comment by laranhee** [comment]
간단하지 않네요 ㅠㅠ

Oauth 플로우를 구현하면 App Directory(마켓같은)에 노출되는 조건을 충족하지 못하도, 설치 url을 통해 다른 워크스페이스에 설치가 가능한 걸로 보이는데 제가 이해한게 맞을까요?

**Comment by giyeonkwon** [comment]
네 저도 그렇게 이해하고 있어요. 일단 2단계를 완료하면 다른 workspace에서 사용할 수 있는거는 분명한거 같아요. 사용방법은 조금 찾아봐야할거 같네요. ㅎㅎ 우선 메뉴얼 하나씩 읽어보고 있습니다. 이해되는대로 업데이트 다시 할게요.

**Comment by laranhee** [comment]
감사합니다 🙇
항상 무리는 하지 않으셨으면 좋겠습니다 (_ _)
